server/client-config/routes.js

```diff
@@ -12,4 +34,23 @@ module.exports = app => {
     res.type('js')
     res.send(response)
   })
+
+  app.get('/ping', nocache, async (req, res) => {
+    try {
+      const exists = await DBExists()
+      if (!exists) {
+        sendResult(res, 404, 'database error')
+        return
+      }
+      const { Body } = await s3.getObject({ Key: 'healtcheck-file' }).promise()
+      if (!Body) {
+        sendResult(res, 404, 's3 connection error')
+        return
+      }
```

Comment:
With this setup, if the database it down, you don't get to know anything about S3. As and when more checks are added, this could be a problem. Another way to structure this could be to check all the things and then report the status of each in the response (with an HTTP error status if any one of them has an error).